robot/Cumulus/resources/ManageHouseHoldPageObject.py

```diff
@@ -0,0 +1,77 @@
+from cumulusci.robotframework.pageobjects import BasePage
+from cumulusci.robotframework.pageobjects import pageobject
+from BaseObjects import BaseNPSPPage
+from NPSP import npsp_lex_locators
+from logging import exception
+
+@pageobject("Custom", "ManageHousehold")
+class ManageHouseholdPage(BaseNPSPPage, BasePage):
+
+    def _is_current_page(self):
+        """
+        Waits for the current page to be a Data Import list view
+        """
+        self.selenium.wait_until_location_contains("/list",timeout=60, message="Records list view did not load in 1 min")
+        self.selenium.location_should_contain("DataImport__c",message="Current page is not a DataImport List view")
+
```

Comment:
Looks like u forgot to change this from DataImport page to manage households page

Reply:
Thanks Sravani. Corrected it